### ISSUE TITLE
FormlyBootstrapModule clobbers root FormlyModule config

### ISSUE THREAD

**Issue by goosmurf** [ngx-formly/ngx-formly#1535]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**

When FormlyBootstrapModule is imported into lazily loaded modules it clobbers the FormlyConfig declared in the root module.

**Expected behavior**

When lazily loaded module imports FormlyModule.forChild({}) it should extend the root config, instead of overwriting it. This is based on my understanding of the behaviour of FormlyMaterialModule as demonstrated in https://github.com/ngx-formly/ngx-formly/issues/1306.

**Minimal reproduction of the problem with instructions**

https://stackblitz.com/edit/angular-sgf5uc

We expect "asdfasdf field is required" validation message from the root config to show up when input field on /route1 is left empty. It doesn't.

Comment out FormlyBootstrapModule from route1.module.ts and the expected behaviour appears.

**Comment by aitboudad** [comment]
fixed in `5.0.4`

**Comment by goosmurf** [comment]
Confirmed fixed. Many thanks for the quick response!
